app/controllers/concerns/user_controllers.rb

```diff
@@ -0,0 +1,30 @@
+class UsersController < ApplicationController
+  def show
+    @user = User.find_by id: (params[:id])
+  end
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+    if @user.save
+      log_in @user
+      flash[:success] = t ".flash.success"
+      redirect_to @user
+    else
+      render :new
+    end
+  end
+
+  def edit
+    @user = User.find_by(params[:id])
+  end
+
+  private
+    def user_params
+      params.require(:user).permit:name, :email, :password,
```

Comment:
```suggestion
      params.require(:user).permit :name, :email, :password,
```
Hàm này dùng để làm j em?